Android/src/main/kotlin/com/applicaster/react/RNBroadcasterManager.kt

```diff
@@ -0,0 +1,28 @@
+package com.applicaster.react
+
+import android.content.Intent
+import androidx.localbroadcastmanager.content.LocalBroadcastManager
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactContextBaseJavaModule
+import com.facebook.react.bridge.ReactMethod
+import com.facebook.react.bridge.ReadableMap
+
+class RNBroadcasterManager(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext){
+
+    val TAG = "RNBroadcasterManager"
+    val SEND_BROADCAST_ACTION = "send_broadcast_from_rn"
+    val EVENT_NAME = "event_name"
+    val EVENT_PROPERTIES = "event_properties"
+
+    override fun getName(): String {
+        return TAG
+    }
+
+    @ReactMethod
+    fun sendBroadcastEvent(eventName: String, properties: ReadableMap) {
```

Comment:
we discussed to have two methods one with eventName and properties and second with eventName only.

Reply:
@snooky23 It appears that It is not possible to overload functions in Javascript... any suggestions?

Reply:
Added a null check for properties